server/src/modules/user/user.controller.ts

```diff
@@ -441,19 +442,18 @@ export class UserController {
     @ApiOperation({ summary: 'Returns the maps submitted by the local user' })
     @ApiOkPaginatedResponse(MapDto, { description: 'Paginated list of submitted maps' })
     getSubmittedMaps(
-        @LoggedInUser('id') _userID: number,
-        @Query() _query?: UserMapSubmittedGetQuery
+        @LoggedInUser('id') userID: number,
+        @Query() query?: UserMapSubmittedGetQuery
     ): Promise<PaginatedResponseDto<MapDto>> {
-        return void 0;
-        //return this.usersService.GetSubmittedMaps(userID, query.skip, query.take, query.search, query.expand);
+        return this.usersService.getSubmittedMaps(userID, query.skip, query.take, query.search, query.expand);
     }
 
-    // I dunno what this last one is, check old API!
     @Get('/maps/submitted/summary')
+    @ApiOkPaginatedResponse(MapSummaryDto, { description: 'The users map statusFlags and their count' })
     @ApiOperation({ summary: 'Returns the summary of maps submitted by the local user' })
     @ApiOkResponse({ description: 'Summary of maps submitted by the local user' })
-    getSubmittedMapsSummary(@LoggedInUser('id') _userID: number) /*: Promise<who knows???> */ {
-        return void 0;
+    getSubmittedMapsSummary(@LoggedInUser('id') userID: number): Promise<PaginatedResponseDto<MapSummaryDto>> {
+        return this.usersService.getSubmittedMapsSummary(userID);
```

Comment:
Sorry, I just realised this is the wrong return type. The old API doesn't return a paginated response, just an array of MapSummaryDtos. So quite a bit of logic needs to change in this commit. Should've noticed earlier, sorry!